### PR TITLE
Wyatt/eth2 GitHub action

### PR DESCRIPTION
**Pull request by Spacesai1or**
## Description

Reverted initial merge (#3892) since we have an active RC

This PR needs to be merged after #3885

### CODE REVIEW

**Review by GregTheGreek — DISMISSED** [review]

**Review by frankiebee — DISMISSED** [review]
🤙

File: packages/web3-eth2-beaconchain/test/schemaMethods.test.ts [hunk 1]
```diff
@@ -135,7 +135,7 @@ it('getValidatorBalances', async () => {
     expect(response[0]).toMatchObject(expectedResponse)
 })
 
-it('getEpochCommittees', async () => {
+xit('getEpochCommittees', async () => {
```
Comment (GregTheGreek) [inline packages/web3-eth2-beaconchain/test/schemaMethods.test.ts:138]: can you make an issue so we can address this in the future?

File: packages/web3-eth2-beaconchain/tsconfig.json [hunk 2]
```diff
@@ -1,21 +1,70 @@
...
-        "web3-eth2-beaconchain": ["."]
-    }
-  }
-}
\ No newline at end of file
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```
Comment (GregTheGreek) [inline packages/web3-eth2-beaconchain/tsconfig.json]: consider targeting a specific version incase future versions are breaking. i.e es2020

Comment (Spacesai1or) [reply packages/web3-eth2-beaconchain/tsconfig.json]: Updated to `ES2020`

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**